lib/ts/recipe/recipeRouter/index.tsx

```diff
@@ -21,10 +23,35 @@ export abstract class RecipeRouter {
             return undefined;
         }
 
-        const component = routeComponents.find((c) => c.matches());
-        if (component !== undefined) {
-            return component;
+        const dynamicLoginMethods = Multitenancy.getInstanceOrThrow().getLoadedDynamicLoginMethods();
+        const possiblyEnabledRecipes = {
+            thirdpartyemailpassword: {
+                enabled: dynamicLoginMethods?.["thirdparty"].enabled && dynamicLoginMethods["emailpassword"].enabled,
```

Comment:
I don't think the second case in the ADR examples works as expected.

Reply:
its working fine